spec/features/manage_cmr/groups/create_group_spec.rb

```diff
@@ -60,7 +60,7 @@
           end
 
           within '.group-form' do
-            VCR.use_cassette('edl/urs/multiple_users', record: :new_episodes) do
```

Comment:
is VCR use_cassette needed for each of these, I think probably only the Submit is were it makes a network call.

Reply:
I'm referring to lines 44-60 in this file

Reply:
I think so because there is a test on lines 71-81, that checks these the content of these members.
<img width="566" alt="Screen Shot 2022-10-07 at 1 58 23 PM" src="https://user-images.githubusercontent.com/63017465/194620558-b475c2ef-6ebd-4bfa-ba75-77800891d3c7.png">